app/controllers/api_controller.rb

```diff
@@ -49,4 +50,47 @@ def page_preview
     }
   end
 
+  # Send content for a page
+  def pages_content
+    page = Page.find_by(id: params[:id])
+    unless page
+      return render json: { errors: [ { detail: "Unable to find page" } ] }, status: 404
+    end
+
+    # Get content
+    content = params[:content]
+    unless content
+      return render json: { errors: [ { detail: "content paramter not provided" } ] }, status: 400
+    end
+
+    # Check page content
+    page.custom_html(content)
+    changes = PollPage.perform(page: page)
+
+    puts changes.inspect
+
+    if changes
+      render json: changes
```

Comment:
For consistency with the other response, returning something like `{ changed: true, status: 'Page snapshotted', snapshot: changes }` might be good.